clarify the membership request (#505)
open-telemetry/community: .github/ISSUE_TEMPLATE/membership.md

---

<!-- Please remember to change the title of this issue by replacing

>  <your-GH-handle> with the actual GitHub handle -->

Reviewer: 👍

### GitHub Username

e.g. (at)example_user

### Requirements

- [ ] I have reviewed the community membership guidelines (https://github.com/open-telemetry/community/blob/master/community-membership.md)
- [ ] I have enabled 2FA on my GitHub account. See https://github.com/settings/security
- [ ] I have subscribed to the [Gitter channel](https://gitter.im/open-telemetry/community)
- [ ] I am actively contributing to 1 or more OpenTelemetry subprojects

> - [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines. Among other requirments, sponsors must be approvers or maintainers of at least one repository and not being affiliated with a single company

Reviewer 2: ```suggestion
- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines. Among other requirements, sponsors must be approvers or maintainers of at least one repository in the organization and not both affiliated with the same company
```
Otherwise it sounds like one must have at least two jobs to be allowed to sponsor someone ;)

- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors

<!-- Replace (at) with the `@` sign -->

- (at)sponsor-1
- (at)sponsor-2

> **New!**: Each sponsor should reply to this issue with the comment "*I support*".

Reviewer: Do we need to mention "New"? People apply once for membership (ignoring the rare case of rejection), so they probably don't care what is new vs old.

Author: this is new for sponsors. Some sponsors do not know what to do, some was doing it differently for a while now. So I wanted to emphasize it. Are you and @arminru strongly against?

Reviewer 2: I'm not sure if "repeat sponsors" will notice it more likely with the "New!" but I'm not strongly opposed, feel free to keep it.

Reviewer 3: people had been using reactji to support :) which I feel also is fine

Author: reactji do not produce a notification in inbox. Harder to track whether there were any updates on the issue

> Please remember, it is an applicant responsibility to notify sponsor about this application.

Reviewer 2: ```suggestion
Please remember, it is an applicant's responsibility to notify their sponsors about this application.
```

or

```suggestion
Please remember, it is an applicant's responsibility to get their sponsors' confirmation before submitting the request.
```
